openshift/ci-operator/update-ci.sh

```diff
@@ -44,8 +44,8 @@ $CURDIR/generate-ci-config.sh knative-$VERSION 4.6 true false > ${CONFIG}__46.ya
 $CURDIR/generate-ci-config.sh knative-$VERSION 4.7 true false > ${CONFIG}__47.yaml
 $CURDIR/generate-ci-config.sh knative-$VERSION 4.8 true false > ${CONFIG}__48.yaml
 $CURDIR/generate-ci-config.sh knative-$VERSION 4.9 true false > ${CONFIG}__49.yaml
-$CURDIR/generate-ci-config.sh knative-$VERSION 4.10 true false > ${CONFIG}__410.yaml
-$CURDIR/generate-ci-config.sh knative-$VERSION 4.11 true false true > ${CONFIG}__411-tls.yaml
+$CURDIR/generate-ci-config.sh knative-$VERSION 4.11 true false true > ${CONFIG}__tls.yaml
```

Comment:
Please see my comment at https://github.com/openshift/release/pull/32080#issuecomment-1240278252
I think we don't need this file `${CONFIG}__tls.yaml`